cypress/integration/desktop/desktopVisitorCanSeeInformationView.feature.js

```diff
@@ -0,0 +1,67 @@
+describe("visitor can see information view", () => {
```

Comment:
would you mind putting in this here as well? 
```/* eslint-disable no-undef */```

Reply:
added